apps.awesim.org/apps/bc_desktop/owens.yml

```diff
@@ -1,6 +1,13 @@
 ---
 title: "Owens Desktop"
 cluster: "owens"
+description: |
+  This app will launch an interactive desktop on one or more compute nodes. You
+  will have full access to the resources these nodes provide. This is analogous
+  to an interactive batch job.
```

Comment:
Note that this bit up at the top is the default which is what's currently being shown.